### PR TITLE
list the existing/official team roles

### PR DESCRIPTION
**Pull request by afeld**
Part of https://github.com/18F/tts-tech-portfolio/issues/262.

### CODE REVIEW

File: roles.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Roles
+
+Roles assigned to individuals are mostly those created externally and filled by members of the Tech Portfolio, rather than created within our team.
```
Comment (its-a-lisa) [inline roles.md]: I don't think we've really discussed or agreed on whether or not we're going to create internal roles for within the team as well and that was one of the things I was trying to get at with https://github.com/18F/tts-tech-portfolio/issues/262

Comment (afeld) [reply roles.md]: Removed the line.

File: roles.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Roles
```
Comment (its-a-lisa) [inline roles.md]: Would also add role for the IaaS responsibility

Comment (afeld) [reply roles.md]: Good catch!

**Pull request merged**